src/v/kafka/server/snc_quota_manager.cc

```diff
@@ -13,10 +13,50 @@
 #include "kafka/server/logger.h"
 #include "prometheus/prometheus_sanitize.h"
 
+#include <fmt/core.h>
+
 using namespace std::chrono_literals;
 
 namespace kafka {
 
+template<std::integral T>
+inoutpair<T> operator+(const inoutpair<T>& lhs, const inoutpair<T>& rhs) {
```

Comment:
I would rename this struct `ingress_egress_stats`. `inoutpair` is not immediately obvious to code readers whats the purpose of the struct

Reply:
it works with more than just `quota_t` so it's not always a "quota"; "stats" looks confusing to me as well. I agree the name is somewhat unclear though... some ideas may be
* `ingress_egress` - then the member names should be changed too IMO
* 'in_out` - would that be good enough?